src/client/pool/__init__.py

```diff
@@ -35,113 +35,109 @@
-            after all tasks are completed. Minimum — 0, maximum — 259200 seconds (three days).
-            Use it if:
-                * Your data processing is close to real time.
-                * You need an open pool where you upload tasks.
-                * Dynamic overlap is enabled in the pool (dynamic_overlap_config).
+        project_id: The ID of the project containing the pool.
+        private_name: The pool name. It is visible to the requester and is not visible to Tolokers.
+        may_contain_adult_content: The presence of adult content.
+        reward_per_assignment: Payment in US dollars for a Toloker for completing a task suite. For cents, use the dot as a separator.
+            If the pool `type` is `REGULAR`, the minimum payment per task suite is $0.005. For other pool types, you can set the `reward_per_assignment` to zero.
+        assignment_max_duration_seconds: Time limit to complete one task suite.
+            Take into account loading a page with a task suite and sending responses to the server. It is recommended that you set at least 60 seconds.
+            Tasks not completed within the limit are reassigned to other Tolokers.
+        defaults: Default settings that are applied to new tasks in the pool.
+        will_expire: The UTC date and time when the pool is closed automatically, even if not all tasks are completed.
+        private_comment: A comment about the pool. It is visible to the requester and is not visible to Tolokers.
+        public_description: The pool description. If pool's `public_description` is not set, then project's `public_description` is used.
+        public_instructions: The pool instructions for Tolokers. If pool's `public_instructions` is not set, then project's `public_instructions` is used.
+        auto_close_after_complete_delay_seconds: The pool remains open after all tasks are completed during the specified time in seconds.
+
+            Use non zero value if:
+            * You process data in real time.
+            * The pool must stay open so that you can upload new tasks.
+            * Dynamic overlap is enabled in the pool.
+
+            Allowed range: from 0 to 259200 seconds (3 days). The default value is 0.
         dynamic_pricing_config: The dynamic pricing settings.
-        auto_accept_solutions: Whether tasks must be checked manually:
-            * True - Automatic task acceptance (manual checking isn't necessary).
-            * False - The requester will check the tasks.
-        auto_accept_period_day: Optional[int]
+        auto_accept_solutions:
+            * True — Responses from Tolokers are accepted or rejected automatically based on some rules.
+            * False — Responses are checked manually. Time reserved for checking is limited by the `auto_accept_period_day` parameter.
+                Learn more about [non-automatic acceptance](https://toloka.ai/en/docs/guide/concepts/offline-accept) in the Requester's guide.
```

Comment:
We no longer use the `Requester's guide` term in the docs. Please replace this sentence with `Learn more about [non-automatic acceptance](https://toloka.ai/en/docs/guide/concepts/offline-accept).`